### PR TITLE
(PC-14556) feat(ProfileDeletion): Add new info screen for suspended users

### PR DESCRIPTION
**Pull request by aliraiki**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-14556

## Checklist

I have:

- [x] Made sure the title of my PR follows the [convention](1) `($jira) $type($scope): $summary`.
- [x] Made sure my feature is working on the relevant real / virtual devices (native and web).
- [x] Written **unit tests** native (and web when implementation is different) for my feature.
- [x] Added a **screenshot** for UI tickets.
- [x] Made sure translations file is up to date (`yarn translations:extract`)

## Screenshots

| iOS | Android | Mobile - Chrome | Desktop - Chrome | Desktop - Safari |
| --: | ------: | --------------: | ---------------: | ---------------: |
|   Simulator Screen Shot - iPhone 13 - 2022-04-28 at 16 03 35  |         |                 | Capture d’écran 2022-04-28 à 16 03 20       |                  |

[1]: https://github.com/pass-culture/pass-culture-app-native/blob/master/doc/standards/pr-title.md
[2]: https://www.notion.so/passcultureapp/R-solution-de-probl-mes-sur-les-bugs-5dd6df8f6a754e6887066cf613467d0a

### CODE REVIEW

File: src/features/auth/suspendedAccount/SuspendedAccount/SuspendedAccount.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+import { ProfileDeletionIllustration } from 'ui/svg/icons/ProfileDeletionIllustration'
+import { getSpacing, Spacer, Typo } from 'ui/theme'
+import { useCustomSafeInsets } from 'ui/theme/useCustomSafeInsets'
+
+export const SuspendedAccount = () => {
+  const { goBack, canGoBack } = useGoBack(...homeNavConfig)
+  const { top } = useCustomSafeInsets()
+  const { data: settings } = useAppSettings()
+
+  useEffect(() => {
```
Comment (kopax-polyconseil) [inline src/features/auth/suspendedAccount/SuspendedAccount/SuspendedAccount.tsx:26]: `useFocusEffect(useCallback(() => {},[]))` ?

Sinon le screen est monté une fois et si on repasse dessus, alors le useEffect ne sera pas executé

File: src/features/auth/suspendedAccount/SuspendedAccount/index.tsx [hunk 2]
```diff
@@ -0,0 +1 @@
+export * from './SuspendedAccount'
```
Comment (kopax-polyconseil) [inline src/features/auth/suspendedAccount/SuspendedAccount/index.tsx:1]: Il est vraiment utile ce fichier ?

File: src/locales/fr/messages.po [hunk 3]
```diff
@@ -2666,11 +2674,15 @@ msgstr "Ton code postal | Profil"
 
 #: src/features/auth/signup/AccountCreated/AccountCreated.tsx
 msgid "Ton compte a été activé !"
-msgstr "Ton compte a été activé !"
+msgstr "Ton compte a été activé !<<<<<<< HEAD"
```
Comment (kopax-polyconseil) [inline src/locales/fr/messages.po:2677]: Probleme ici de merge

File: src/locales/fr/messages.po [hunk 4]
```diff
@@ -2368,8 +2375,9 @@ msgstr "Revenir à l'accueil"
 #: src/features/offer/components/useReportOfferModalContent.tsx
 #: src/ui/components/achievements/components/GenericAchievement.tsx
 msgid "Revenir à l'étape précédente"
-msgstr "Revenir à l'étape précédente"
+msgstr "Revenir à l'étape précédente<<<<<<< HEAD"
```
Comment (kopax-polyconseil) [inline src/locales/fr/messages.po:2378]: Probleme ici de merge

File: src/features/auth/suspendedAccount/SuspendedAccount/index.tsx (hunk 2 above)
Comment (aliraiki) [reply src/features/auth/suspendedAccount/SuspendedAccount/index.tsx]: C'était pour avoir des imports plus jolis mais je l'ai supprimé, je verrais si on n'en a besoin après

**Review by kopax-polyconseil — APPROVED** [review]

**Pull request merged**